### PR TITLE
Recent Contributors section - refactor followup

### PR DESCRIPTION
**Pull request by Rishabh570**
No issue opened for this. It is a step towards making the codebase more flexible and maintainable.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by Rishabh570** [comment]
@jywarren A step towards our flexible queryTime in recent contribs logic, making the section of the code flexible and segregated so that future improvements can be (more) easily installable and maintainable. Please take a look :)

File: src/utils/contribsUtil/storeAllContribsInDb.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+				let splicedRepos = res.splice(0, 20);
+				splicedRepos.map(function mappingToEachRepo(Repo, i) {
+				  let promise = fetchRepoContributorsUtil.fetchRepoContributorsUtil(org, Repo)
+								.then(function gotRepoContributorsInStorage(contributors) {
+								  if(contributors!=undefined && contributors.length>0) {
+									contributors.map((contributor, i)=> {
+									  if(!contributorSet.has(contributor.login)) {
+										contributorSet.add(contributor.login);
+										AllContributors.push(contributor);
+									  }
```
Comment (jywarren) [inline src/utils/contribsUtil/storeAllContribsInDb.js]: Looks super nice! Some extreme indentation here, though, did you want to adjust that? Thanks!

**Comment by jywarren** [comment]
This looks really nice. Do you think it starts to need more documentation in the README about code structure yet, or not quite yet? Like in this README: https://github.com/publiclab/PublicLab.Editor/

This can be merged either way! Thanks!!!

**Comment by Rishabh570** [comment]
Yes, a more descriptive readme would surely help everyone who's trying to involve in the project...I have that as a to-do in the [project](https://github.com/publiclab/community-toolbox/projects/1). I am thinking of something like showing the flowchart for general control flow and then explaining what every part of the code does.

File: src/utils/contribsUtil/storeAllContribsInDb.js (hunk 1 above)
Comment (Rishabh570) [reply src/utils/contribsUtil/storeAllContribsInDb.js]: Done!

**Comment by Rishabh570** [comment]
@jywarren This is ready to merge I think, please take a look :)

**Comment by jywarren** [comment]
Awesome!!!!

**Pull request merged**